recipe_scrapers/plugins/exception_handling.py

```diff
@@ -0,0 +1,65 @@
+import functools
+import logging
+
+from recipe_scrapers.settings import settings
+
+from ._interface import PluginInterface
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+class ExceptionHandlingPlugin(PluginInterface):
+    """
+    Plugin that is used only if settings.EXCEPTION_HANDLING is set to True.
+
+    The outer-most plugin and decorator.
+
+    If ANY of the methods listed raises ANY kind of exception, silence it
+    and return the respective value from settings.ON_EXCEPTION_RETURN_VALUES
+
+    If settings.EXCEPTION_HANDLING is set to False this plugin is ignored and
+    does nothing. (In other words exceptions won't be handled and will bubble up
+    to program's explosion. Left to the end-user to handle them on his own).
+    """
+
+    run_on_hosts = ("*",)
+    run_on_methods = (
+        "title",
+        "total_time",
+        "yields",
+        "image",
+        "ingredients",
+        "instructions",
+        "ratings",
+        "reviews",
+        "links",
+        "language",
+        "nutrients",
+    )
+
+    @classmethod
+    def run(cls, decorated):
+        @functools.wraps(decorated)
```

Comment:
You're probably considered this during implementation, so apologies for asking what is probably a naive question: is there a way we could avoid plugin implementors needing to include this 'internal' `@functools.wraps` and decorated function?

Reply:
I'd rather leave the plugins template like this for now. More explicit. It gives a lot of (probably still not enough) freedom to do whatever people may come up with. Be it nutrients, ingredients grouping, translations etc. Let's see how other devs will use the functionality: what problems will be reported and what features requested.